### PR TITLE
[JENKINS-65398] Terminology update

### PR DESCRIPTION
**Pull request by Vlatombe**
Remove usage of master, slave, whitelist, blacklist

- [X] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your master branch!
- [X] Ensure that the pull request title represents the desired changelog entry
- [X] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by Vlatombe** [comment]
ping @jglick since you cut the last release

File: src/main/java/hudson/slaves/CommandConnector.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ private Object readResolve() {
     @Override
     public CommandLauncher launch(String host, TaskListener listener) throws IOException, InterruptedException {
         // no need to call ScriptApproval.using here; CommandLauncher.launch will do that
-        return new CommandLauncher(new EnvVars("SLAVE", host), command);
+        return new CommandLauncher(new EnvVars("SLAVE", host,"AGENT", host), command);
```
Comment (jglick) [inline src/main/java/hudson/slaves/CommandConnector.java:63]:
```suggestion
        return new CommandLauncher(new EnvVars("SLAVE", host, "AGENT", host), command);
```

File: src/test/java/hudson/slaves/CommandLauncherTest.java [hunk 2]
```diff
@@ -132,28 +132,28 @@ private String createWorkspace() throws IOException {
         return tempDir.getAbsolutePath();
     }
 
-    public DumbSlave createSlave(String command, String workspacePath) throws Exception {
-        DumbSlave slave;
+    public DumbSlave createAgent(String command, String workspacePath) throws Exception {
+        DumbSlave agent;
```
Comment (jglick) [inline src/test/java/hudson/slaves/CommandLauncherTest.java:136]: kinda pointless :grin:

**Review by jglick — APPROVED** [review]

File: src/test/java/hudson/slaves/CommandLauncherTest.java (hunk 2 above)
Comment (Vlatombe) [reply src/test/java/hudson/slaves/CommandLauncherTest.java:136]: Yeah I know, also harmless so why not do it ;)

**Pull request merged**